spec/semantic_conventions.md

```diff
@@ -7,83 +7,87 @@ operations used by applications. These conventions are used to populate the `att
+| `reranker.query`                       | String                      | `"How to format timestamp?"`                                               | Query parameter of the reranker                                                     |
+| `reranker.top_k`                       | Integer                     | 3                                                                          | Top K parameter of the reranker                                                     |
+| `retrieval.documents`                  | List of objects<sup>†</sup> | `[{"document.id": "1", "document.score": 0.9, "document.content": "..."}]` | List of retrieved documents                                                         |
+| `session.id`                           | String                      | `"26bcd3d2-cad2-443d-a23c-625e47f3324a"`                                   | Unique identifier for a session                                                     |
+| `tag.tags`                             | List of strings             | ["shopping", "travel"]                                                     | List of tags to give the span a category                                            |
+| `tool.description`                     | String                      | `"An API to get weather data."`                                            | Description of the tool's purpose and functionality                                 |
+| `tool.json_schema`                     | JSON String                 | `"{'type': 'function', 'function': {'name': 'get_weather'}}"`              | The json schema of a tool input                                                     |
+| `tool.name`                            | String                      | `"WeatherAPI"`                                                             | The name of the tool being utilized                                                 |
+| `tool.parameters`                      | JSON string                 | `"{ 'a': 'int' }"`                                                         | The parameters definition for invoking the tool                                     |
+| `tool_call.function.arguments`         | JSON string                 | `"{'city': 'London'}"`                                                     | The arguments for the function being invoked by a tool call                         |
+| `tool_call.function.name`              | String                      | `"get_current_weather"`                                                    | The name of the function being invoked by a tool call                               |
+| `tool_call.id`                         | string                      | `"call_62136355"`                                                          | The id of the by a tool call (useful when thare are more than one at the same time) |
```

Comment:
```suggestion
| `tool_call.id`                         | string                      | `"call_62136355"`                                                          | The id of the called tool (useful when thare are more than one at the same time) |
```

Reply:
The `call` is the principal object here, not the `tool`, e.g. the same tool can be called with different arguments.

Reply:
added tool.id